src/frontends/tensorflow_common/src/op/binary_op.cpp

```diff
@@ -52,6 +52,7 @@ template OutputVector translate_binary_op<Multiply>(const NodeContext& node);
 template OutputVector translate_binary_op<Mod>(const NodeContext& node);
 template OutputVector translate_binary_op<NotEqual>(const NodeContext& node);
 template OutputVector translate_binary_op<Power>(const NodeContext& node);
+template OutputVector translate_binary_op<PRelu>(const NodeContext& node);
```

Comment:
can we have layer tests for this?
I recommend to add `PRelu` and `HSwish` in separate PR because our current approach is strict that is not to add translators without tests.

Reply:
There is no HSwish or PRelu in normal tensorflow by documentation.
However, PRelu could be constructed with keras, but it gets decomposed while representing in tensorflow.
It seems these two won't be used by TF FE in the nearest future.
I ask to postpone addressing this comment.
@rnugmanx prepares layer tests for most the operations supported in this PR. HSwish and PRelu will be there for sure.

Reply:
Ok, let us proceed. But this is not acceptable to add new functionality into the common part without tests. You know - this bad practice has already created many problems.

Reply:
Agree. Will establish proper way of layer testing tflite and then will always introduce layer test for enabled operation.